src/defaultPhrases.js

```diff
@@ -0,0 +1,176 @@
+const closeDatePicker = 'Close';
+const focusStartDate = 'Focus on start date';
```

Comment:
We're gonna run this through a copy person before we upgrade react-dates in Airbnb, so maybe let's leave it til then?